SmartDeviceLink/SDLHapticInterface.h

```diff
@@ -0,0 +1,32 @@
+//
+//  SDLHapticInterface.h
+//  SmartDeviceLink-iOS
+//
+//  Copyright © 2017 smartdevicelink. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "SDLConnectionManagerType.h"
+
+@class SDLManager;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol SDLHapticInterface <NSObject>
+
+/**
+ Initializes haptic interface. After initializing the application must call updateInterfaceLayout to process the UIWindow. Application must update later view changes in the window by sending SDLDidProjectionViewUpdate  notification.
+ 
+  @param window UIWindow to initialize
```

Comment:
I'd be careful with this; we are merely utilizing the UIWindow, not initializing it.